src/styles/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the image `star.svg` is present in the `images` directory. This is necessary for the background image to display correctly.